### PR TITLE
Upgrade Compute disks to >= 0.11

### PR DESCRIPTION
**Pull request by Everlag**
Further operation changes have been made to match our testing
requirements. Specifically, the `ClientHelper` now supports zone
operations and `get_global_operation` has been migrated.

### CODE REVIEW

File: lib/fog/compute/google/models/disk.rb [hunk 1]
```diff
@@ -57,15 +57,15 @@ def zone_name
         # auto_delete can only be applied to disks created before instance creation.
         # auto_delete = true will automatically delete disk upon instance termination.
         def get_object(writable = true, boot = false, device_name = nil, auto_delete = false)
-          mode = writable ? "READ_WRITE" : "READ_ONLY"
+          writable ? mode = "READ_WRITE" : mode = "READ_ONLY"
```
Comment (icco) [inline lib/fog/compute/google/models/disk.rb]: oh man, that's ugly. Could you expand this out into an `if` statement? I somewhat regret this stylistic choice.

File: lib/fog/compute/google/models/disk.rb [hunk 2]
```diff
@@ -45,7 +45,7 @@ def destroy(async = true)
           requires :name, :zone
 
           data = service.delete_disk(name, zone_name)
-          operation = Fog::Compute::Google::Operations.new(:service => service).get(data.body["name"], data.body["zone"])
+          operation = Fog::Compute::Google::Operations.new(:service => service).get(data.name, data.zone)
```
Comment (icco) [inline lib/fog/compute/google/models/disk.rb]: In other PRs you've been referencing these as symbols. Is there a difference here?

Comment (Everlag) [inline lib/fog/compute/google/models/disk.rb]: Ah, thanks for the consistency check, fixed.

File: lib/fog/compute/google/models/disk.rb (hunk 1 above)
Comment (Everlag) [inline lib/fog/compute/google/models/disk.rb]: Done. I'm usually not a fan of ternaries.

**Pull request merged**